client/src/components/related_items/OutfitListCard.jsx

```diff
@@ -31,10 +32,14 @@ const OutfitListCard = ({card, onDelete , handleRelatedItemClick}) =>{
             <div className='OutfitListCard'>
                     <div onClick={()=>{handleRelatedItemClick(card.id)}}>    
                         <img className="image" src={`${photoURL}`} /> 
-                        <p className="category">{card.category}</p>
-                        <h3 className='productName'>{card.name}</h3>
-                        <h4 className='price'>${card.default_price}</h4>
-                        <p>{card.rating}</p>
+                        <div className='cardText'>
+                            <p className="category">{card.category}</p>
+                            <h3 className='productName'>{card.name}</h3>
+                            <h4 className='price'>${card.default_price}</h4>
+                            <div className='stars'>
+                                <Stars productId={card.id} />
+                            </div>
+                        </div>
```

Comment:
Adding more detail to div's is a good practice and will allow us to fine tune our program. Good change